stdlib/public/core/BridgeObjectiveC.swift

```diff
@@ -166,16 +166,17 @@ public func _bridgeAnythingToObjectiveC<T>(_ x: T) -> AnyObject {
 }
 
 @_silgen_name("")
```

Comment:
What is this doing?

Reply:
https://github.com/apple/swift/pull/13005#discussion_r151886972

Reply:
Oof, so we're using the usableFromInline-ness but also the ability to have such a function without a body. I guess that can't change for now.